docs/streams/stream-examples/realtime-updates.md

```diff
@@ -144,25 +308,25 @@ if __name__ == '__main__':
         client.on_change(COLLECTION_NAME, callback=callback_fn, timeout=15)
 
     # Step3: Subscribe to receive documents in realtime (PUSH model)
-    print(f"\n3. SUBSCRIBE_COLLECTION: region: {GLOBAL_URL},  collection: {COLLECTION_NAME}")
+    print(f"\n3. SUBSCRIBE_COLLECTION: region: {URL},  collection: {COLLECTION_NAME}")
     rt_thread = threading.Thread(target=create_callback)
     rt_thread.start()
     time.sleep(10)
     print(f"Callback registered for collection: {COLLECTION_NAME}")
 
-    # Step4: Subscribe to receive documents in realtime (PUSH model)
-    print(f"\n4. INSERT_DOCUMENTS: region: {GLOBAL_URL},  collection: {COLLECTION_NAME}")
+    # Step4: Subscribe to receive documents in real-time (PUSH model)
+    print(f"\n4. INSERT_DOCUMENTS: region: {URL},  collection: {COLLECTION_NAME}")
     client.insert_document(COLLECTION_NAME, document=data)
 
     # Step5: Wait to close the callback.
     print("\n5. Waiting to close callback")
     rt_thread.join(2)
 
-    # Step6: Delete data.
-    print(f"\n6. DELETE_DATA: region: {GLOBAL_URL}, collection: {COLLECTION_NAME}")
+    # Step 6: Delete data.
+    print(f"\n6. DELETE_DATA: region: {URL}, collection: {COLLECTION_NAME}")
```

Comment:
On the printout, I see play.macrometa.... as the region. That does not seem right; it isn't a region.